### ISSUE TITLE
convos exited with code 0

### ISSUE THREAD

**Issue by trymbill** [convos-chat/convos#885]
**Describe the bug**
I'm trying to run Convos with Docker and been struggling getting past the first user being registered. I've tried a bunch of things (permissions, UID/GID, different reverse proxy settings, different browsers, etc) but can't find the source of the issue. Every time I try and 'Sign up' the initial user, I get this:

<img width="849" alt="Screenshot 2024-03-07 at 22 32 46" src="https://github.com/convos-chat/convos/assets/658727/da5bc7b8-f140-444e-81bd-ea2f0e0c48d5">

Here's the docker-compose.yaml config:
```yaml
convos:
  container_name: "convos"
  image: "ghcr.io/convos-chat/convos:stable"
  user: "1000:100"
  environment:
    - "CONVOS_LOG_LEVEL=trace"
    - "CONVOS_REVERSE_PROXY=1"
    - "CONVOS_REQUEST_BASE=https://my.domain.com/"
  volumes:
    - "/path/to/convos/data:/data"
  restart: "unless-stopped"
  labels:
    - "traefik.enable=true"
    - "traefik.http.routers.convos.tls.certresolver=letsencrypt"
    - "traefik.http.routers.convos.entrypoints=https"
    - "traefik.http.routers.convos.rule=Host(`my.domain.com`)"
    - "traefik.http.services.convos.loadbalancer.server.port=3000"
    - "traefik.backend=convos"
    - "traefik.frontend.rule=Host:my.domain.com"
```

Here's the call to `/api/user/register.json` being made in Postman and a terminal open on the docker container running Convos. As you can see the container exits with `code 0` and nothing in the logs, even on `CONVOS_LOG_LEVEL=trace`, to explain why:

https://github.com/convos-chat/convos/assets/658727/ff9c5964-88da-4bf7-ac5d-ad65328780ab

Any ideas what could be going on here?

**Environment:**
 - OS: Ubuntu
 - Browser: Chrome, Firefox, Arc
 - Version: ghcr.io/convos-chat/convos:stable

**Comment by jberger** [comment]
I reported something perhaps similar via IRC on Jan 31. Though I didn't see exit code 0, I saw 132.

>  it seems that using images from ghcr.io when I login I segfault the pod
exit code 132
which as best as I can understand means "illegal instruction" which I think basically means I'm using the wrong architecture
it looks like the image I pulled say amd64 so I have no idea what is going on
but backing all the way up to convos/convos:v7.15 fixes it

**Comment by asdofindia** [comment]
Could also be an issue with dependencies like in #881

**Comment by jhthorsen** [comment]
What a great bug report! Makes me even more sad that I have no clue what's going on...

Even so, I just bumped some dependencies, so if you have time, please try out https://github.com/convos-chat/convos/pkgs/container/convos/192095675?tag=alpha

I'm also rebuilding the [base docker image](https://github.com/convos-chat/docker-base/actions/runs/8319331499) now, and going to make a new release of Convos afterwards. (Hopefully today)

**Comment by jhthorsen** [comment]
I'm going to close this as a duplicate of #862. Please let me know if you disagree.

**Comment by jhthorsen** [comment]
I'm reopening this issue, since I closed #862 before waiting for feedback from all users.

**Comment by jberger** [comment]
@jhthorsen using Crypt::Argon2 0.22 (which contains https://github.com/Leont/crypt-argon2/pull/14) if we build with `CRYPT_ARGON2_ARCH=NONE` in the environment, it should work.
